types/svg2png/index.d.ts

```diff
@@ -1,10 +1,18 @@
 // Type definitions for svg2png node package 
 // Project: https://github.com/domenic/svg2png
-// Definitions by: hans windhoff <https://github.com/hansrwindhoff>
+// Definitions by: songChengcheng <https://github.com/sccgithub>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function svg2png(srcFile: string, destFile: string, scale: number, cb: (err:Error) => void):void;
-declare function svg2png(srcFile: string, destFile: string, cb: (err:Error) => void):void;
+/// <reference types="node" />
 
+interface opts {
+  fileName?: string,
+  width?: number,
+  height?: number,
+  url?: string
+}
 
-export = svg2png;
+declare function svg2png(sourceBuffer: Buffer, opts?: opts): Promise<Buffer>;
+export function sync(sourceBuffer: Buffer, opts?: opts): Buffer;
+
+export default svg2png;
```

Comment:
@sccgithub  as @andy-ms  mentioned this is not correct since according to https://github.com/domenic/svg2png/blob/master/lib/svg2png.js#L11 it seems like require("svg2png") is the module.